xcodeproj/internal/bazel_integration_files/create_lldbinit.sh

```diff
@@ -82,6 +82,8 @@ touch "$lldbinit"
 
 readonly required_source='command source ~/.lldbinit-rules_xcodeproj'
 if ! grep -m 1 -q "$required_source" "$lldbinit"; then
```

Comment:
Can you also adjust this, so it only matches when it's a whole line? Otherwise this fix won't fix already corrupt files out there.

Reply:
Yep make sense, let me get that updated